websites/T/Teknofest/metadata.json

```
@@ -0,0 +1,21 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "PXSTY",
+		"id": "571041849326698496"
+	},
+	"service": "Teknofest",
+	"description": {
+		"tr": "TEKNOFEST Havacılık, Uzay ve Teknoloji Festivali",
+		"en": "TEKNOFEST Havacılık, Uzay ve Teknoloji Festivali",
+		"nl": "TEKNOFEST Havacılık, Uzay ve Teknoloji Festivali"
+	},
+	"url": ["teknofest.org"],
```

Site supports www subdomain
```suggestion
	"url": ["teknofest.org","www.teknofest.org"],
```